pages/embedded.vue

```diff
@@ -78,7 +78,7 @@ const { data, error, status } = await useFetch<ApiPoi>(
       geometry_as: 'bbox',
       short_description: false,
     },
-    immediate: !!poiId.value,
+    immediate: !!poiId.value && !poiId.value.includes('_'),
```

Comment:
starts with, not includes ?

Reply:
no i never starts with _.
Address IDs may have the following shapes:
- 57108_ywbbdg
- 57108_651651_afdzf554